pyanaconda/storage_utils.py

```diff
@@ -242,6 +251,12 @@ def verify_partition_sizes(storage, constraints, report_error, report_warning):
                            % {'mount': mount, 'size': size,
                               'productName': productName})
 
+    for (mount, size) in constraints[STORAGE_REQ_PARTITION_SIZES].items():
+        if mount in filesystems and filesystems[mount].size < size:
+            report_error(_("Your %(mount)s partition is less than "
+                           "%(size)s which is lower than required.")
+                         % {'mount': mount, 'size': size})
```

Comment:
This sentence sounds weird. Could it be rephrased in something like this?
`Your %(mount)s partition size is lower than required %(size)s.`

Reply:
This message is based on the warning above.

Reply:
Fixed.

Reply:
Looks great now. Thanks a lot!